src/SearchWrapper.cpp

```diff
@@ -197,9 +197,102 @@ parseQuery(Rcpp::List & query)
 		}
 	    }
 	}
-
+      
+#else
+      
+      //check for valueRangeProcessors
+      if(query.containsElementNamed("VRP")){ 
+        Rcpp::List processors=query["VRP"];
+        
+        for( Rcpp::List::iterator it = processors.begin(); it != processors.end(); ++it ) {
+          Rcpp::List list=it.operator*();
+          string type=Rcpp::as<std::string>(list["type"]);
+          
+          if(!type.compare("proc.num")){ 
+            int aVal=list["value"];
+            if(list.containsElementNamed("check.str")){
+              string aStr=Rcpp::as<std::string>(list["check.str"]);
+              
+              bool aPrefix=true;
+              
+              Rcpp::StringVector flagVec;
+              if (list.containsElementNamed("flags")) {
+                flagVec = list["flags"];
+                for (Rcpp::StringVector::iterator itr = flagVec.begin(); itr != flagVec.end(); ++itr) {
+                  std::string aflag = Rcpp::as<std::string>(*itr);
+                  if ((!aflag.compare("RP_SUFFIX")) || (!aflag.compare("RP_REPEATED")))
+                        aPrefix=false;
+                }
+              }
+              
+              Xapian::NumberValueRangeProcessor *num_proc=new Xapian::NumberValueRangeProcessor(aVal,aStr,aPrefix);
+              queryparser.add_valuerangeprocessor(num_proc);
+            }
+            else{
+              Xapian::NumberValueRangeProcessor *num_proc=new Xapian::NumberValueRangeProcessor(aVal);
+              queryparser.add_valuerangeprocessor(num_proc);
+            }
+          }
+          if(!type.compare("proc.str")){
+            int aVal=list["value"];
+            if(list.containsElementNamed("check.str")){
+              string aStr=Rcpp::as<std::string>(list["check.str"]);
+              bool aPrefix=true;
+              
+              Rcpp::StringVector flagVec;
+              if (list.containsElementNamed("flags")) {
+                flagVec = list["flags"];
+                for (Rcpp::StringVector::iterator itr = flagVec.begin(); itr != flagVec.end(); ++itr) {
+                  std::string aflag = Rcpp::as<std::string>(*itr);
+                  if ((!aflag.compare("RP_SUFFIX")) || (!aflag.compare("RP_REPEATED")))
```

Comment:
Sorry, I think I've got confused between prefix and suffix :-(

The default for RXapian users should be the same no matter what, meaning it's a prefix unless you pass `RP_SUFFIX`. So you were right, except that you don't want to handle `RP_REPEATED` with Xapian 1.2.

Don't worry about displaying an error for `c('RP_REPEATED')`, because that's really Xapian's responsibility to complain if that combination of flags doesn't make sense.